src/WebJobs.Script/Description/Proxies/ProxyClientExecutor.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Azure.AppService.Proxy.Client.Contract;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.Description
+{
+    internal static class ProxyClientExecutor
```

Comment:
Nit: can we make this class wrap IProxyClient and then just pass this around instead?  So the methods would become instance methods. 

IProxyClient realClient = ...;
var client = new ProxyClient(realCLient); 
client.Execute(Request, logger)